r/NEWS.md

```diff
@@ -19,12 +19,65 @@
 
 # arrow 0.16.0.9000
 
+## Feather v2
+
+This release includes support for version 2 of the Feather file format.
+Feather v2 features full support for all Arrow data types, and it allows
```

Comment:
Mention also that it fixes the 2GB per-column limitation for large amounts of string data. We will include this in blog post too